include/flatbuffers/flatbuffers.h

```diff
@@ -1216,18 +1216,30 @@ class Verifier FLATBUFFERS_FINAL_CLASS {
            size_t _max_tables = 1000000)
     : buf_(buf), end_(buf + buf_len), depth_(0), max_depth_(_max_depth),
       num_tables_(0), max_tables_(_max_tables)
+    #ifdef FLATBUFFERS_TRACK_VERIFIER_BUFFER_SIZE
```

Comment:
The variable doesn't have to be guarded really, the most important is the code in Check/Verify.